### ISSUE TITLE
Additional hashtags on edited posts are not found

### ISSUE THREAD

**Issue by hoergen** [friendica/friendica#6001]
### Expected behavior
when editing a post and adding a hashtag. The hashtag should be found by the search

### Actual behavior
The hashtag aka the post is not found.

### Steps to reproduce the problem
Create a post, submit it and then edit it again and add a hashtag.

### Friendica version you encountered the problem
git, develope, mariadb ,php7

**Comment by lapo-luchini** [comment]
Uh, so editing [a message where a parenthesis was parsed as part of the tag](https://f.lapo.it/display/6a1cc041-235c-d15b-c676-44f014072323) wouldn't help me.
Good to know. :)
(I dislike "unnecessary message edits")

**Comment by iworkinpixels** [comment]
Likewise, if you edit a post and *remove* a hashtag, that hashtag remains on the post forever.

**Comment by forenta** [comment]
Well, at least I was able to find a workaround for that issue. Friendica doesn´t ping the Fediverse about the update.

### Bug Description

If you add hashtags to existing Friendica posts and someone else on another instance is opening that hashtag, you will not find the post.

### Steps to Reproduce



1.  Be friendicauser@friendinstance.xyz, get a follower from mastodon.social so that mastodon.social knows about your posts.
2.  Create a post on Friendica
3.  Add the hashtag #​goddamnbug to your post.
4. Open https://mastodon.social/tags/goddamnbug

Actual Result:

![grafik](https://user-images.githubusercontent.com/2451238/235730394-984a73fc-4642-46f1-ad76-1b62baa05cf9.png)

Expected Result:

See the post tagged with #​goddamnbug.

### Workaround

1. Edit the post you've tagged with #​goddamnbug, edit something - even adding a ZWSP is fine.
2. Hang your laundry
3. Open https://mastodon.social/tags/goddamnbug
4. 🎉

### Platform Info

Friendica Version: 2023.06-dev

**Comment by MrPetovan** [comment]
What if you hang your laundry in the first case?

**Comment by forenta** [comment]
Wouldn´t fix the problem that other instances don´t get any update.

**Comment by MrPetovan** [comment]
Ok, I just wanted to make sure it wasn't an outbound notifying task timing issue where you would look for the tag immediately after adding the tag on edition compared to waiting a bit after you fake-edit the post later.

**Comment by forenta** [comment]
Nope. Tried this with posts I´ve tagged weeks ago. Works, but at least mastodon.social takes 2-200 seconds for the update.

**Comment by MrPetovan** [comment]
2-200 seconds delay in which case? Regular case or workaround?

**Comment by forenta** [comment]
Workaround. In the regular case, the added hashtag will _never_ appear on mastodon.social. During peak hours, it generally takes a minute for posts from remote instances to appear on mastodon.social.

**Comment by MrPetovan** [comment]
Thank you for the elaboration!
